content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -67,6 +67,24 @@ This status means that logs are enabled but you haven't specified which containe
 
 2. To configure the Agent to collect from other containers, set the `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` environment variable to `true`.
 
+
+## Docker log collection from file issues
+
+When collecting Docker container logs from file the Agent will fall back on collection from Docker socket if it cannot read from the directory where Docker container logs are stored (`/var/lib/docker/containers` on Linux). In some circumstances the Datadog agent may fail to collect log from file. This can be diagnosed by checking the Logs Agent status and looking for an file type entry showing an error like:
+
+```text
+    - Type: file
+      Identifier: ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834
+      Path: /var/lib/docker/containers/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834-json.log
+      Status: Error: file /var/lib/docker/containers/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834/ce0bae54880ad75b7bf320c3d6cac1ef3efda21fc6787775605f4ba8b6efc834-json.log does not exist
+```
+
+This status means that the agent is unable to find a log file for a given container, to resolve this issue check that the folder containing Docker containers logs is correctly exposed to the Datadog Agent container. On Linux it corresponds to  `-v /var/lib/docker/containers:/var/lib/docker/containers:ro` on the command line starting the Agent container whereas on Windows it translates to `-v c:/programdata/docker/containers:c:/programdata/docker/containers:ro`. Note that the directory relatively to the underlying host may be different due to spectific config of the Docker daemon, this is not an issue pending a correct Docker volume mapping. E.g. `-v /data/docker/containers:/var/lib/docker/containers:ro` if the Docker data directory has been relocated to `/data/docker` on the underlying host.
+
+If log are collected but single appears to be split please check that the Docker daemon is using the [JSON logging driver][#your-containers-are-not-using-the-json-logging-driver].
+
+The log collection from file is activated when the envionment variable `DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE` is set to `true`. However some containers may still be tailed from the Docker socket despite this setting. Only containers started after the activation of the option will have their logs collected from file. This situation is typically found during Datadog Agent upgrade from a version that was not offering this feature. If desired it is possible to force all containers, including older ones, to have their logs collected from file by setting the envionment variable `DD_LOGS_CONFIG_DOCKER_CONTAINER_FORCE_USE_FILE` to `true`. This might lead to some duplicated logs when the transition occurs.
```

Comment:
```suggestion
Log collection from file is activated when the environment variable `DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE` is set to `true`. However, some containers may still be tailed from the Docker socket despite this setting. Only containers started after the activation of the option will have their logs collected from file. This situation is typically found when the Datadog Agent is upgraded from a version that was not offering this feature. If desired, it is possible to force all containers (including older ones) to have their logs collected from file by setting the environment variable `DD_LOGS_CONFIG_DOCKER_CONTAINER_FORCE_USE_FILE` to `true`. This may lead to some duplicated logs when the transition occurs.
```